zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/GroupQueryController.java

```diff
@@ -55,6 +65,26 @@ public ResponseEntity<GroupSearchQueryResponse> searchGroups(
         .fold(RestErrorMapper::mapProblemToResponse, this::search);
   }
 
+  @PostMapping(
+      path = "/{groupKey}/users/search",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE},
+      consumes = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<UserSearchResponse> searchUsersInGroup(
+      @PathVariable final long groupKey,
+      @RequestBody(required = false) final UserSearchQueryRequest query) {
+    final var groupMemberKeys =
+        groupServices
+            .search(
+                SearchQueryBuilders.groupSearchQuery().filter(f -> f.groupKey(groupKey)).build())
```

Comment:
This is limited by the page size, but we need all results here.

Reply:
That's a good point, thanks!